src/tribol/mesh/MfemData.hpp

```diff
@@ -1554,6 +1553,8 @@ class MfemSubmeshData
   /**
    * @brief Get the gap vector on the parent-linked boundary submesh
    *
+   * @note This is stored as a dual vector, meaning the shared DOFs must be summed over all ranks to obtain their value.
```

Comment:
This is stored as a mfem dual vector, meaning they expect the shared DOFs to be summed over all ranks to obtain the appropriate values. or something like that.